JavaScript/main.js

```diff
@@ -0,0 +1,12 @@
+let total = 0;
+
+function adopt(fee) {
+  total += fee;
+  alert(total);
+}
+
+function inspect(name, breed, fee) {
+  alert(name);
```

Comment:
It may be more preferable as a user to see one alert with a more nicely formatted message. 
See JS Template Literals:  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals